### PR TITLE
Add image_ops in Addons

### PR DESCRIPTION
**Pull request by vinayvr11**
Add  solarize image_ops in Addons. https://github.com/tensorflow/addons/issues/1226
https://github.com/tensorflow/addons/issues/1353

### CODE REVIEW

**Comment by abhichou4** [comment]
> @abhichou4
> Add solarize image_ops in Addons.

Looks almost ready. Look at how to add typing in contribution guidelines.

**Comment by vinayvr11** [comment]
> > @abhichou4
> > Add solarize image_ops in Addons.
> 
> Looks almost ready. Look at how to add typing in contribution guidelines.

Finally it's done.

File: tensorflow_addons/image/solarize_ops.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+""" This module is used to invert all pixel values above a threshold
```
Comment (autoih) [inline tensorflow_addons/image/solarize_ops.py]: Thanks @vinayvr11. Can you also add license like this? https://github.com/tensorflow/addons/blob/4eeea6f972b18aff7df60202c7386b35a455a832/tools/releases/release_linux.sh#L2

File: tensorflow_addons/image/solarize_ops_test.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+"""Test of solarize_ops"""
```
Comment (autoih) [inline tensorflow_addons/image/solarize_ops_test.py]: Thanks @vinayvr11, please also add license here, too.

Comment (vinayvr11) [reply tensorflow_addons/image/solarize_ops_test.py]: Sure

**Comment by vinayvr11** [comment]
Done

**Comment by vinayvr11** [comment]
@abhichou4 : I just added the new script of posterize.

File: tensorflow_addons/image/posterize_ops_test.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+@test_utils.run_all_in_graph_and_eager_modes
+class PosterizeOpsTest(tf.test.TestCase, parameterized.TestCase):
+    """PosterizeOpsTest class to test the working of
+    methods images"""
+
+    def test_posterize(self):
+        """ Method to test the posterize technique on images """
+        if tf.executing_eagerly():
+            image = tf.constant(
```
Comment (fsx950223) [inline tensorflow_addons/image/posterize_ops_test.py]: How about ```image = tf.ones([4,4],dtype=tf.uint8)*255```

Comment (vinayvr11) [reply tensorflow_addons/image/posterize_ops_test.py]: Sure i will update this.

Comment (vinayvr11) [reply tensorflow_addons/image/posterize_ops_test.py]: Done

**Comment by Squadrick** [comment]
Create separate PRs for each op

**Pull request closed without merging**